cfbs/index.py

```diff
@@ -235,6 +250,26 @@ def save(self, data=None):
     def _module_is_in_build(self, module):
         return module["name"] in (m["name"] for m in self["build"])
 
+    def validate_added_module(module):
+        """Try to help the user with warnings in appropriate cases"""
+
+        name = module["name"]
+        if name.startswith("./") and name.endswith(".cf"):
+            assert os.path.isfile(name)
+            if not _has_autorun_tag(name):
+                log.warning("No autorun tag found in policy file: '%s'" % name)
```

Comment:
no current objection to these messages, I think it's clear. Possibly could be _more_ specific and say Tag the bundle(s) in FILENAME ...

Reply:
The policy filename is there, in the first log message. I don't think I should repeat it.